mixed/flink/flink-common/pom.xml

```diff
@@ -143,6 +143,12 @@
                 </exclusion>
             </exclusions>
         </dependency>
+        <dependency>
+            <groupId>org.apache.flink</groupId>
+            <artifactId>flink-metrics-dropwizard</artifactId>
+            <version>${flink.version}</version>
+            <scope>provided</scope>
+        </dependency>
```

Comment:
Why do we need this dependency?

Reply:
The Iceberg connector's operator(IcebergStreamingWriter.writerMetrics) needs this dependency.